internal/oid/oid.go

```diff
@@ -12,3 +12,8 @@ var RawSNPReport = asn1.ObjectIdentifier{1, 3, 9901, 2, 1}
 // RawTDXReport is the root OID for the raw TDX report extensions
 // used by the aTLS issuer and validator.
 var RawTDXReport = asn1.ObjectIdentifier{1, 3, 9901, 2, 2}
+
+// WorkloadSecretOID is the root OID for the workloadSecretID report
+// extension, added to the mesh certificates to allow verification
+// and authorization based on the workloadSecretID.
+var WorkloadSecretOID = asn1.ObjectIdentifier{1, 3, 9901, 2, 3}
```

Comment:
`1.3.9901` is the "as you like it" prefix, right? Should we take a different next number to distinguish report extensions from claim extensions? Don't we already have an extension number for claims somewhere?